Exception {msg: , stackTrace: [foo(): Exception, getException(): Exception, main(): none]}
